dashboard/src/components/Config/AppRepoList/AppRepoList.tsx

```diff
@@ -144,25 +200,13 @@ function AppRepoList({
           {!errors.fetch && (
             <>
               <LoadingWrapper loaded={!isFetching}>
-                <h3>Global Repositories</h3>
-                <p>
-                  Global repositories are available for all Kubeapps users.{" "}
-                  {kubeappsCluster &&
-                    (kubeappsCluster !== cluster || namespace !== kubeappsNamespace) && (
-                      <>
-                        Administrators can go to the{" "}
-                        <Link to={app.config.apprepositories(kubeappsCluster, kubeappsNamespace)}>
-                          {kubeappsNamespace}
-                        </Link>{" "}
-                        namespace to manage them.
-                      </>
-                    )}
```

Comment:
The link to the `kubeapps` namespace is no longer needed since it's not a requirement to edit these repos.